zhmccli/_cmd_nic.py

```diff
@@ -107,6 +107,17 @@ def nic_show(cmd_ctx, cpc, partition, nic):
 @click.option('--device-number', type=str, required=False,
               help='The device number to be used for the new NIC. '
               'Default: auto-generated')
+@click.option('--ssc-management-nic', type=bool, required=False,
+              help='Indicates that this NIC should be used as a management '
+              'NIC for Secure Service Container to access the web interface.')
+@click.option('--ssc-ip-address-type', type=str, required=False,
+              help='"ipv4, ipv6, linklocal or dhcp.')
+@click.option('--ssc-ip-address', type=str, required=False,
+              help='IP Address of the SSC management web interface.')
+@click.option('--ssc-mask-prefix', type=str, required=False,
+              help='Network Mask of the SSC management NIC.')
```

Comment:
Add to help text: "Only applicable to and required for NICs of ssc type partitions when ssc-ip-address-type is ipv4 or ipv6."